openedx/features/caliper_tracking/transformers/third_party_transofrmers.py

```diff
@@ -0,0 +1,38 @@
+"""
+Transformers for all the Third-Party events
+"""
+
+
+def oppia_exploration_completed(current_event, caliper_event):
+    """
+    The server emits an oppia.exploration.loaded event
+    when an Oppia exploration component is shown in the LMS.
```

Comment:
Please correct the doc string according to the event.

Reply:
Done